app/components/provider_relationship_permissions_list.html.erb

```diff
@@ -17,14 +17,3 @@
     <% end %>
   </ul>
 <% end %>
-
-<% if show_view_applications_only_section? %>
-  <% providers_that_can_only_view_applications = providers_that_can('view_applications_only') %>
-  <p class="govuk-!-margin-bottom-5 govuk-body">
-    <% if providers_that_can_only_view_applications.size > 1 %>
-      <%= training_provider.name %> has not set up permissions for this partnership yet.
-    <% else %>
-      <%= providers_that_can('view_applications_only').first.name %> can only view applications.
```

Comment:
Do we want to remove this whole section? What if a training provider has set up the permission in such a way that only one provider in the relationship has all the power?

Reply:
I didn't quite understand the relevance of the `size` condition but I agree we want one of two messages depending on whether setup is required or not.
https://github.com/DFE-Digital/apply-for-teacher-training/pull/2730/commits/5ee272d15c3f8f7ef6f4bfb5c23e29afbb274291 should do this.

Reply:
I don't think we need to check the size.
The `at_least_one_active_permission_in_pair` validation on the model means that we can either have 1 or zero providers who can only view application and `show_view_applications_only_section?` already checks if there is a provider who can only see applications (`providers_that_can(:view_applications_only).any?`)

The whole thing LGTM.